docs/changelog/changelog_en.md

```diff
@@ -1,5 +1,11 @@
 # Changelog plentyshopPWA
 
+## v1.6.1
+
+- Added new carousel compoment via Swipper library
+- Created new useCarousel composable holding Carousel logic
+- Minified and purged swiper css files for speed optimization
+
```

Comment:
**todo**: Please add the changelog to the existing `v1.x.x` section instead of creating a new one.